npm-global-without-sudo.md

```diff
@@ -21,6 +21,8 @@ npm config set prefix "${HOME}/.npm-packages"
 Add the following to your `.bashrc`/`.zshrc`:
 
 ```sh
+export XDG_DATA_HOME=${HOME}/.local/share
```

Comment:
We should not do this. That's too opinionated. macOS does not use XDG. We should not force it upon macOS users.

Reply:
Okay. The recent commit addresses this.